rules/sonic-ztp.dep

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
better to remove ```ifeq ($(ENABLE_ZTP), y)``` in sonic-ztp.mk, otherwise if ztp is not enabled, it will report such error.
```
fatal: Not a git repository (or any of the parent directories): .git
```

i found it is difficult to debug such error, there is no clue which file to introduce this error.

Reply:
Fixed and uploaded the patchset.